Formula/vice.rb

```diff
@@ -39,20 +39,19 @@ def install
       "--prefix=#{prefix}",
       "--disable-dependency-tracking",
       "--disable-arch",
-      "--disable-bundle",
       "--enable-external-ffmpeg",
     ]
 
     if build.head?
       configure_flags << "--enable-native-gtk3ui"
-      configure_flags << "--disable-hwscale"
     else
+      configure_flags << "--disable-bundle"
       configure_flags << "--enable-sdlui2"
     end
 
     system "./autogen.sh" if build.head?
     system "./configure", *configure_flags
-    system "make", "install"
+    system "make", "-j", "install"
```

Comment:
No, this is taken care of automatically behind the scenes.

Reply:
ok, that change is made